lib/hooks/useCookieDrop.js

```diff
@@ -0,0 +1,16 @@
+const useCookieDrop = () => {
```

Comment:
useCookieDrop --> useSetCookie

You will also need useGetCookie

Reply:
@salilkapoor writing this again, this extra wrapping is not required. Every JS file is automatically a module in ES6  and this additional wrapping doesn't serve any purpose but increases code side. 

I am merging it but you can fix it as separate PR.v